src/cpp/shared/game/game.h

```diff
@@ -174,25 +179,39 @@ class game {
 	void SetMapSize(int, int);
 	void SetupAdjacentMons();
 	void SetVisibility(int,int,int,int);
-    void SetVisibility_orig(int, int, int, int);
+  void SetVisibility_orig(int, int, int, int);
 
 	void ClaimTown(int,int,int);
-    int GetTownId(int,int);
-
+  int GetTownId(int,int);
+  
 	void NextPlayer();
 	void NextPlayer_orig();
 
 	void PerDay();
 	void PerDay_orig();
 
-    void ResetIronfistGameState();
-    void ShareVision(int sourcePlayer, int destPlayer);
+  void ResetIronfistGameState();
```

Comment:
Is the indentation right for all of these?

Reply:
Those inconsistencies are not reflected in my version of the file; to me, the indentation looks fine.

Reply:
It's probably a tabs vs. spaces issue. Prefer spaces.

Reply:
I'm pretty sure that I got the tabs vs. spaces thing fixed, but I'll check it again.

Reply:
The configuration looks fine for me.

Reply:
Okay, I'll consider this fine.